restli-server/src/main/java/com/linkedin/restli/server/ResourceDefinition.java

```diff
@@ -43,10 +43,20 @@
 
   /**
    * Gets the rest.li resource java class.
+   * By default we assume java resource class name is the same as annotated namespace name,
+   * if that class cannot be found, return null.
    *
    * @return java class for this rest.li resource.
    */
-  Class<?> getResourceClass();
+  default Class<?> getResourceClass() {
+
+    String clazzName = getNamespace() + "." + getName();
+    try {
+      return Class.forName(clazzName);
```

Comment:
This might be incorrect class, no? If the namespace is different from resource class's package. 
I think we can just throw unsupported exception instead

Reply:
ok, that is even simpler.